.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
```

Comment:
Rename the file to `ci.yml` and the workflow to CI.

Reply:
Not sure about the name - it's a bit like having a string called string or a class called Class - but no strong feelings here, I'll adjust.

Reply:
That's the name used in other repos.